natives.json

```diff
@@ -17707,19 +17707,19 @@
 			"build": "1207"
 		},
 		"0xAC2767ED8BDFAB15": {
-			"name": "_0xAC2767ED8BDFAB15",
-			"comment": "",
+			"name": "_DRAIN_PED_HEALTH",
```

Comment:
This should be ``_SET_ENTITY_HEALTH``.
Comment and params should be edited accordingly.